queue/python/queue.py

```diff
@@ -0,0 +1,32 @@
+class Queue:
+    def __init__(self):
+        self.items = []
+
+    def isEmpty(self):
+        return self.items == []
+
+    def enqueue(self, item):
+        self.items.insert(0,item)
+
+    def dequeue(self):
+        return self.items.pop()
+
+    def size(self):
+        return len(self.items)
+    
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpvd7j5r87/queue/python/queue.py
+++ b/tmp/tmpvd7j5r87/queue/python/queue.py
@@ -18,7 +18,7 @@
         return self.items[self.size()-1] 
 
 def main():
-    q=Queue()
+    q = Queue()
     q.enqueue(4)
     q.enqueue('dog')
     q.enqueue(True)
```